### DISCUSSION TITLE
I Can`t Install Osu!Lazer

### DISCUSSION THREAD

**Discussion by RaihanFa64** [ppy/osu#14651] [Q&A] [answered]
Hi :) Can you help me because when I click osu! lazer(install.exe)
a pop up appears: how do you want to open this file?
Please is there someone who can help me?

**Comment by frenzibyte** [comment]
That does not sound like you're in Windows. What OS are you running?

**Reply by RaihanFa64** [reply]
I'm using windows 10 x64 bit

> That does not sound like you're in Windows. What OS are you running?

**Reply by MishterKirby** [reply]
Try redownloading it, as it seems like your copy is corrupt.

**Reply by RaihanFa64** [reply]
Already, is it possible because I don't have framework 5.0 sdk??

**Reply by MishterKirby** [reply]
If you're getting a box like this, It's because Windows doesn't have an association for the file type:
![image](https://user-images.githubusercontent.com/48867925/132272806-31293a08-0c96-41c6-b033-ae6c7d030bcc.png)
This sounds like either your download is corrupt, or your Windows installation is corrupt.

**Reply by RaihanFa64** [reply]
So, what should I do?

**Reply by MishterKirby** [reply]
Try following the instructions here to reset file associations:

https://www.tenforums.com/tutorials/8703-restore-default-file-type-associations-windows-10-a.html

**Reply by peppy** [reply]
what is a vrm file?

**Reply by RaihanFa64** [reply]
> what is a vrm file?

no, it's just an example

**Reply by RaihanFa64** [reply]
I've tried everything but still can't :(

**Comment by RaihanFa64** [comment]
Thank you! for the help it seems I have found the solution Thank you very much! :)

**Reply by MishterKirby** [reply]
It would be helpful if you posted the solution here, in case anyone else experiences the same issue.

**Reply by RaihanFa64** [reply]
> It would be helpful if you posted the solution here, in case anyone else experiences the same issue.

ok :)

**Comment by RaihanFa64** [comment, marked as answer]
So, I suddenly got an idea to solve this problem i.e. I downloaded Opera Browser and re-downloaded osu!Lazer in Opera Browser and opened it in Opera Browser and it worked :)

**Reply by Modsnotworking** [reply]
ok but still how did you fixed it?? like how did you opened it in opera browser

**Reply by RaihanFa64** [reply]
I don't know maybe as @MishterKirby said that the file I downloaded is corrupt
